project/Dependencies.scala

```diff
@@ -44,7 +44,8 @@ object Dependencies {
     * @see https://github.com/guardian/subscriptions-frontend/pull/363#issuecomment-186190081
     */
   val googleDirectoryAPI = Seq(
-    "com.google.apis" % "google-api-services-admin-directory" % "directory_v1-rev53-1.20.0" exclude("com.google.guava", "guava-jdk5"),
+    "com.google.apis" % "google-api-services-admin-directory" % "directory_v1-rev118-1.25.0" exclude("com.google.guava", "guava-jdk5"),
+    "com.google.api-client" % "google-api-client" % "1.30.9", // Required as it fixes https://github.com/googleapis/google-api-java-client/issues/1487
```

Comment:
why not 1.30.10 ? https://mvnrepository.com/artifact/com.google.api-client/google-api-client/1.30.10

Reply:
My initial (loosely-applied) logic was to specify the minimal dependency update needed, in order to avoid breaking functionality. However, 1.30.9 -> 1.30.10 is a patch update by semver standards - I see what makes more sense is to get the version with the most fixes. Updated, thank you for spotting it!